Themes/default/css/index.css

```diff
@@ -211,10 +211,12 @@ body#help_popup {
 #likes li {
 	clear: both;
 	padding: 1px 0;
+	display: flex;
+	align-items: center;
 }
 #likes .avatar {
-	max-height: 45px;
-	max-width: 45px;
+	height: 50px;
+	width: 50px;
```

Comment:
i don't understand where this magic numbers come from.
i would expect to had a min height and possible fit content in width in combination of a grid layout.

Reply:
why grid when it's just a list? you are trying to over complicate 3 columns when the only issue was inconsistent size of avatars.

Reply:
They aren't magic numbers. I just liked 50px better than 45px.

As for why I used flex rather than grid, that was because flex did the job I needed it to do: aligning everything to the centre on the vertical axis, while also making it easy to justify the date string to the right on the horizontal axis.

Reply:
This may be overwritten by an inline <head> declaration that sets width and height with !important...